### ISSUE TITLE
Notification on new incoming transaction

### ISSUE THREAD

**Issue by BitDesert** [Nault/Nault#246]
It would be cool if you could get a browser notification for incoming transactions.

- Natrium does that too for mobile
- Probably not doable when the wallet is closed without a third-party integration
- Should not instantly prompt for the permission (annoying!)
- Section in configuration

Text could be something like "You received 0.12345 NANO".

**Comment by aleksre** [comment]
Agreed. Web Push API + ServiceWorker is probably the best way to do this. I created a branch that added a ServiceWorker and PWA support to Nault a while back, but never actually completed it. I can have a look at that branch again.

For the server side, we need to use a push service. More info here: https://developers.google.com/web/fundamentals/push-notifications/how-push-works#who_and_what_is_the_push_service

This will allow Nault to push OS native notifications even when it is not running on Windows, Mac, Linux and Android
